test/e2e/targets/azureeventhubs/main.go

```diff
@@ -193,9 +292,9 @@ var _ = Describe("Azure Event Hubs target", func() {
 
 					// NOTE: The payload will be a stringified version of the CloudEvent
```

Comment:
This shouldn't be true anymore, right? The message should be serializable to `cloudevent.Event` based on the change proposed in this PR.